build.properties

```diff
@@ -94,7 +94,7 @@ phpunit.sparql_url = sparql://${sparql.host}:${sparql.port}/
 # -------------------------
 
 # Apache Solr version to use.
-solr.version = 6.2.1
+solr.version = 6.3.0
```

Comment:
Yes older versions are removed from the download page. I have been looking at a way to automatically derive the latest version, but it doesn't seem to be simple, they don't have a 'latest' alias on their FTP site. So we'll have to update it manually for the moment.